### DISCUSSION TITLE
Color per Workspace

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#819] [Ideas]
### What feature would you like to see?

Similar to #163, it would be nice to be able to differentiate the workspace in use by the accent of the browser. It would be nice to have different colors for different workspaces for faster feedback.

### What platform are you seeing the problem on?

macOS

### Additional context

When I open a new Workspace, I would like it to be a long-lived "profile" of sorts. to which I'd like to be able to differentiate by container: #163 and by color.
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/499_

**Comment by KiKaraage** [comment]
Adding to the suggestion, I would love a button that redirect to **Settings > Look and Feel** so we can change the color scheme **if Zen supported different themes/color schemes per workspace** in the future.
It can be called either **Customize Workspace**, or **Set Workspace Theme**.
![Zentheming Suggestion 2](https://github.com/user-attachments/assets/d1633f12-29cb-4081-a90b-62dd127ee329)
![Zentheming Suggestion 2b](https://github.com/user-attachments/assets/d42344fc-c562-486b-aabf-0b3aec205547)



cc @gcarrarom

**Comment by blakegearin** [comment]
I believe this can be closed as it seems implemented. Right click > "Change Theme Colors"

**Comment by erutan** [comment]
This is intuitive if you're used to using Zen, but I'm trying it out and it wasn't intuitive for me to set the workspace color outside of the workspace settings.  It's a "beta" feature layered on top of the main application so that's fine, but it really should be an option when you name a space and choose an emoji for it.

**Reply by angrybacon** [reply]
I agree with the above comment, and also wonder whether it could be sync' across devices as well. I expected it to be the case and since I want the same experience across devices, I'd prefer having the exact same color codes.

**Comment by minerop5000** [comment]
I also think this discussion can be closed.
The coloring feature is no longer in beta and feature request (from previous comments) should be separate discussions.
